backend/src/modules/tournament-tree/core/domain/TournamentTree.ts

```diff
@@ -5,19 +5,21 @@ import { EntityIdGenerator } from '../../../../shared/core/application/EntityIdG
 import { DomainCommandResult } from '../../../../shared/core/domain/DomainCommandResult';
 import { CurrentTimeProvider } from '../../../../shared/core/CurrentTimeProvider';
 import { TournamentTreeWasCreated } from './event/TournamentTreeWasCreated';
+import { TournamentTeamId } from './TournamentTeamId';
 
 export class TournamentTree {
   readonly tournamentTeams: TournamentTeam[];
   readonly tournamentTreeArray: FightingTeamsGroup[];
   readonly tournamentId: string;
+  firstPlaceTeamId: string | undefined = undefined;
```

Comment:
Ta zmienna jest nieużywana.

Reply:
ooooo, usunąłem nie to co trzeba i przestała być!
Dzięki za uwage. Cofnę błąd